iOS/MobileStarter/react-app/src/App.tsx

```diff
@@ -77,6 +77,7 @@ function App() {
             rpcInterfaces: getSupportedRpcs(),
             notifications: new AppToolAssistanceNotificationManager(),
             authorizationClient: new ITMAuthorizationClient(),
+            tileAdmin: { gpuMemoryLimits: "aggressive" }
```

Comment:
I don't feel that this change should be in this PR. I'm guessing it slipped in accidentally.

Reply:
Oops! Will remove.

Reply:
Removed